src/modules/client/mixins/menuItemsMixin.js

```diff
@@ -0,0 +1,234 @@
+import { mapGetters } from 'vuex';
+import get from 'lodash/get';
+import Customers from '@api/Customers';
+import {
+  CLIENT_ADMIN,
+  AUXILIARY,
+  PLANNING_REFERENT,
+  AUXILIARY_ROLES,
+  COACH_ROLES,
+  HELPER,
+} from '@data/constants';
+import { defineAbilitiesFor } from '@helpers/ability';
+
+export const menuItemsMixin = {
+  data () {
+    return {
+      customerActiveRoutes: {
+        planning: { open: false },
+        customers: { open: false },
+        administrative: { open: false },
+        teams: { open: false },
+      },
+      coachActiveRoutes: {
+        courses: { open: false },
+        planning: { open: false },
+        customers: { open: false },
+        teams: { open: false },
+        billing: { open: false },
+        pay: { open: false },
+        exports: { open: false },
+        configuration: { open: false },
+      },
+      auxiliaryActiveRoutes: {
+        planning: { open: false },
+        customers: { open: false },
+        administrative: { open: false },
+        teams: { open: false },
+      },
+    }
+  },
+  computed: {
+    ...mapGetters({ company: 'main/company', clientRole: 'main/clientRole' }),
+    customer () {
+      return this.isHelper ? this.$store.getters['customer/getCustomer'] : null;
+    },
+    hasContracts () {
+      return this.loggedUser && this.loggedUser.customers && this.loggedUser.customers.length > 0 &&
+        this.loggedUser.customers[0].contracts && this.loggedUser.customers[0].contracts.length > 0;
+    },
+    hasBillingAssistance () {
+      return get(this.loggedUser, 'company.billingAssistance');
+    },
+    isAdmin () {
+      return CLIENT_ADMIN === this.clientRole;
+    },
+    isAuxiliaryWithCompany () {
+      return [AUXILIARY, PLANNING_REFERENT].includes(this.clientRole);
+    },
+    isHelper () {
+      return this.clientRole === HELPER;
+    },
+    isAuxiliary () {
+      return AUXILIARY_ROLES.includes(this.clientRole);
+    },
+    isCoach () {
+      return COACH_ROLES.includes(this.clientRole);
+    },
+    routes () {
+      let routes = [];
+      const ability = defineAbilitiesFor(this.clientRole, null, this.company);
+
+      if (this.isHelper) routes = this.customerRoutes;
+      else if (this.isCoach) routes = this.coachRoutes;
+      else if (this.isAuxiliary) routes = this.auxiliaryRoutes;
+
+      return routes
+        .map(r => (this.isHelper ? r : { ...r, children: r.children.filter(c => ability.can('read', c.name)) }))
+        .filter(r => r.children ? r.children.length : ability.can('read', r.name))
+    },
+    activeRoutes () {
+      if (this.isHelper) {
+        return {
+          planning: { open: false },
+          customers: { open: false },
+          administrative: { open: false },
+          teams: { open: false },
+        };
+      } else if (this.isCoach) return this.coachActiveRoutes;
```

Comment:
en effet tu as raison pour le collapsibleClosing mais du coup ici aussi tu peux retourner this.customerActivesRoutes pour etre coherent